pkg/liqonet/iptables/iptables.go

```diff
@@ -429,6 +429,68 @@ func (h IPTHandler) removeChainsPerCluster(clusterID string) error {
 	return nil
 }
 
+// EnsurePostroutingRules makes sure that the postrouting rules for a given cluster are in place and updated.
+func (h IPTHandler) EnsurePostroutingRules(isGateway bool, tep *netv1alpha1.TunnelEndpoint) error {
```

Comment:
`isGateway` flag is not useful anymore. It was used when the iptables rules were configured also on the nodes by the `liqo-route` operator. We do not touch anymore the iptables rules of the nodes, so we can drop the flag.